docs/documentation/faq.md

```diff
@@ -43,4 +43,31 @@ without an update:
 ```
 
 Although you might consider using `EventSources`, to handle reconciliation triggering in a smarter
-way. 
\ No newline at end of file
+way. 
+
+### Q: How can I run an operator without cluster scope rights?
+
+By default, JOSDK requires access to CRDs at cluster scope. You may not be granted such
```

Comment:
Is this true? By default we should not require access to CRDs just CRs.

Reply:
I guess, it's not true anymore but the javadoc hasn't been updated when the change was made, so it should be updated as well.